semgrep-core/tests/generate_test_matrix.py

```diff
@@ -12,14 +12,17 @@
 
 
 FEATURES = ["dots", "equivalence", "metavar", "misc"]
+
+VERBOSE_REGEXP_SYNTAX = "OCaml Syntax: \"=~/<regexp>/\""
+
 VERBOSE_FEATURE_NAME = {
-    "dots": "Ellipsis",
-    "equivalence": "Equivalences",
-    "metavar": "Metavariables",
+    "dots": "Wildcard Matches (...)",
```

Comment:
I'm glad we're improving these! The naming was a bit confusing/inconsistent. Although, we should keep these names consistent with how we describe them in the docs: https://github.com/returntocorp/semgrep/blob/develop/docs/pattern-features.md. Looks like that covers "Ellipsis Operator", "Equivalences", "Metavariables", and "Deep Expression Operator."

Reply:
@mschwager yes, I agree that consistency is important- thanks for bringing this up! I am happy to update the docs to match these, too, if you think that makes sense. What I was trying to do here was not so much re-name the operators (ellipsis are ellipsis after all) but instead re-frame the cheatsheet's headings as "the problem the user is trying to solve", rather than "the operator they need", which they may not know the name of yet